android/java/org/chromium/chrome/browser/rewards/BraveRewardsCreatorPanelFragment.java

```diff
@@ -0,0 +1,355 @@
+            });
+        }
+    }
+
+    @Override
+    public void onLargeIconReady(Bitmap icon) {
+        SetFavIcon(icon);
+    }
+
+    @Override
+    public void OnGetExternalWallet(int errorCode, String externalWallet) {
+        int walletStatus = BraveRewardsExternalWallet.NOT_CONNECTED;
+        if (!TextUtils.isEmpty(externalWallet)) {
+            try {
+                BraveRewardsExternalWallet mExternalWallet =
+                        new BraveRewardsExternalWallet(externalWallet);
+                walletStatus = mExternalWallet.getStatus();
+            } catch (JSONException e) {
+                Log.e("BraveRewards", e.getMessage());
+            }
+        }
+        @PublisherStatus
+        int pubStatus = mBraveRewardsNativeWorker.GetPublisherStatus(currentTabId_);
+        setPublisherNoteText(pubStatus, walletStatus);
+    }
+
+    @SuppressLint("ClickableViewAccessibility")
+    private void setPublisherNoteText(@PublisherStatus int pubStatus, int walletStatus) {
+        String notePart1 = "";
+        String walletType = BraveRewardsNativeWorker.getInstance().getExternalWalletType();
+        if (walletStatus == BraveRewardsExternalWallet.NOT_CONNECTED) {
+            if (pubStatus == BraveRewardsPublisher.CONNECTED
+                    || pubStatus == BraveRewardsPublisher.UPHOLD_VERIFIED
+                    || pubStatus == BraveRewardsPublisher.BITFLYER_VERIFIED
+                    || pubStatus == BraveRewardsPublisher.GEMINI_VERIFIED) {
+                Log.e("BraveRewards", "User is unverified and publisher is verified");
+            } else {
+                notePart1 = getResources().getString(
+                        R.string.brave_ui_site_banner_unverified_notice_text);
+                Log.e("BraveRewards", "User is unverified and publisher is unverified");
+            }
+        } else {
+            if (pubStatus == BraveRewardsPublisher.NOT_VERIFIED) {
+                notePart1 = getResources().getString(
+                        R.string.brave_ui_site_banner_unverified_notice_text);
+                Log.e("BraveRewards", "User is verified and publisher is unverified");
+            } else if (pubStatus == BraveRewardsPublisher.CONNECTED
+                    || (pubStatus == BraveRewardsPublisher.UPHOLD_VERIFIED
+                            && !walletType.equals(BraveWalletProvider.UPHOLD))
+                    || (pubStatus == BraveRewardsPublisher.BITFLYER_VERIFIED
+                            && !walletType.equals(BraveWalletProvider.BITFLYER))
+                    || (pubStatus == BraveRewardsPublisher.GEMINI_VERIFIED
+                            && !walletType.equals(BraveWalletProvider.GEMINI))) {
+                notePart1 = getResources().getString(
+                        R.string.brave_ui_site_banner_different_verified_notice_text);
+                Log.e("BraveRewards",
+                        "User is verified and publisher is verified but not with the same provider");
```

Comment:
nit: remove if not for verification and similar logs

Reply:
@SergeyZhukovsky wanted all these logs to be removed and if wanna retain it then i think this should be a debug log than error.

Reply:
yeah, I don't think they bring any benefits to us on user's devices, it's better to have them in debug configuration only or put an assert there if some condition isn't valid.

Reply:
If this is not common case (happens rarely) I would probably keep it as it won't clutter the log, but may help with debugging this issue.

Reply:
converted debug log and added TAG.